src/main/java/io/jenkins/plugins/sprp/YAML_FlowDefinition.java

```diff
@@ -146,23 +152,30 @@ public FlowExecution create(FlowExecutionOwner owner, TaskListener listener,
         return refSpecs;
     }
 
-    /*
-
-    Bit-bucket generated following two similar refspecs:
-    refs/heads/abhishekg1128/readmemd-edited-online-with-bitbucket-1529079381853:refs/remotes/origin/PR-1
-    refs/heads/master:refs/remotes/upstream/master
-
-    And GitHub generates following two similar refspecs in single string"
-    +refs/pull/3/head:refs/remotes/origin/PR-3 +refs/heads/master:refs/remotes/origin/master
-
-    */
-    private String getBranchName(String[] refSpecsArray){
+    /**
+     * Bit-bucket generates following two similar refSpecs for pull request:
+     * {@code refs/heads/abhishekg1128/readmemd-edited-online-with-bitbucket-1529079381853:refs/remotes/origin/PR-1}
+     * {@code refs/heads/master:refs/remotes/upstream/master}
+     *
+     * And GitHub generates following two similar refspecs in single string"
+     * {@code +refs/pull/3/head:refs/remotes/origin/PR-3 +refs/heads/master:refs/remotes/origin/master}
+     * {@code getCleanRefSpecs} removes all the trailing spaces and unwanted '+' symbols from refSpecs.
+     *
+     * As branch name can contain {@code '/'}, this function will split the refSpec provided as parameter with
```

Comment:
Not critical but comments should be in the present tense as they are descriptive: "this function splits the refSpec...", and later "it assumes that all the remaining...".